### PR TITLE
Ledger use web USB

### PR DESCRIPTION
**Pull request by arjunu**
Tested on 
- Chrome Version 91.0.4472.106 (Official Build) (x86_64)
- Chrome Version 93.0.4535.0 (Official Build) canary (x86_64)

### CODE REVIEW

File: packages/web/app/lib/web3/ledger-wallet/LedgerConnector.ts [hunk 1]
```diff
@@ -83,33 +85,40 @@ export class LedgerWalletConnector {
...
 
   public getMultipleAccounts = async (
     derivationPaths: string[],
   ): Promise<IDerivationPathToAddress> => {
-    const accounts = await Promise.all(
-      derivationPaths.map(path => this.getMultipleAccountsFromHdPath(path, 0, 1)),
-    );
+    const accounts = [];
+    for (let i = 0; i < derivationPaths.length; i += 1) {
+      accounts.push(await this.getMultipleAccountsFromHdPath(derivationPaths[i], 0, 1));
```
Comment (arjunu) [inline packages/web/app/lib/web3/ledger-wallet/LedgerConnector.ts:120]: it can only perform one operation at a time so we should do it in serial

**Review by ozanmanav-ng — APPROVED** [review]
LGTM

File: packages/web/app/lib/web3/ledger-wallet/LedgerConnector.ts (hunk 1 above)
Comment (desfero) [reply packages/web/app/lib/web3/ledger-wallet/LedgerConnector.ts:120]: let's put a comment about that so we know the reason

**Review by desfero — CHANGES_REQUESTED** [review]

File: packages/web/app/lib/web3/ledger-wallet/LedgerConnector.ts (hunk 1 above)
Comment (arjunu) [reply packages/web/app/lib/web3/ledger-wallet/LedgerConnector.ts:120]: added

**Review by desfero — APPROVED** [review]

**Pull request merged**